### ISSUE TITLE
Progress dialog is not announced by screen readers

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#1501]

**Comment by armhub** [comment]
When automatically logging in with stored credentials, the progress dialog show a focus border. We should remove it from this dialog.
